pyproject.toml

```
@@ -26,6 +26,8 @@ sentence-transformers = {version ="^2.2.2", optional = true}
 evaluate = {version = "^0.4.0", optional = true}
 faiss-cpu = {version = "^1.7.1", optional = true}
 ipywidgets = {version = "^8.1.1", optional = true}
+presidio-analyzer = "^2.2.351"
```

Consider something like this, same pattern for h5py if we need that with presidio-analyzer
```suggestion
presidio-analyzer = {version = "^2.2.351", optional = true} 
```